boards/arm/mimxrt1040_evk/board.cmake

```diff
@@ -5,5 +5,9 @@
 #
 
 board_runner_args(jlink "--device=MIMXRT1042XXX6B")
+board_runner_args(linkserver "--device=MIMXRT1042xxxxB:MIMXRT1040-EVK")
```

Comment:
Verified I was able to flash an RT1040 EVK with the above enablement

Reply:
> Verified I was able to flash an RT1040 EVK with the above enablement

Me too ;)